clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -406,6 +413,8 @@ public StoreIngestionTask(
         new IngestionNotificationDispatcher(notifiers, kafkaVersionTopic, isCurrentVersion),
         amplificationFactorAdapter);
 
+    this.runnableForKillIngestionTasksForMissingSOP = () -> waitForStateVersion(kafkaVersionTopic);
```

Comment:
Can we verify that this triggers ERROR or an exception in OFFLINE->STANDBY?  The original bug hit was because we got stuck on waiting for someone to call StateModelIngestionProgressNotifier.stopConsumption(), a quick glance implies we only call this if we get an exception in that transition.  Looks like kill() errors the resource, but I'm trying to find where/if it triggers that exception case.

Reply:
Walked though in IDE with Sourav, looks good.  We'll call reportKilled in run() if the consumerAction is processed.